mautrix_telegram/__main__.py

```diff
@@ -121,8 +121,6 @@ def prepare_bridge(self) -> None:
         # Note: In upstream this would start all the puppets, but in our fork we want to gracefully start the user puppets
         self.add_startup_actions(User.init_cls(self))
         self.add_startup_actions(Portal.restart_scheduled_disappearing())
-        if self.bot:
-            self.add_startup_actions(self.bot.start())
```

Comment:
Presumably the bot start is still happening once (somewhere else)?

Reply:
Yes, on line 155